README.md

```diff
@@ -0,0 +1,85 @@
+# The yt Project
+
+[![Users' Mailing List](https://img.shields.io/badge/Users-List-lightgrey.svg)](http://lists.spacepope.org/listinfo.cgi/yt-users-spacepope.org/)
+[![Devel Mailing List](https://img.shields.io/badge/Devel-List-lightgrey.svg)](http://lists.spacepope.org/listinfo.cgi/yt-dev-spacepope.org/)
+[![Build Status](https://img.shields.io/travis/yt-project/yt.svg?branch=master)](https://travis-ci.org/yt-project/yt)
+[![Latest Documentation](https://img.shields.io/badge/docs-latest-brightgreen.svg)](http://yt-project.org/docs/dev/)
+[![Data Hub](https://img.shields.io/badge/data-hub-orange.svg)](https://hub.yt/)
+                
+[![yt logo](doc/source/_static/yt_logo.png "yt logo")](http://yt-project.org/)
```

Comment:
Replacing this line with the following seems to make the image render smaller. Feel free to choose a different width :)

`<a href="http://yt-project.org"><img src="doc/source/_static/yt_logo.png" width="300"></a>`